submissions/blessingOluwapelumi/javascript/conditionals/conditionals.js

```diff
@@ -0,0 +1,13 @@
+for (let i = 0; i < 173; i++) {
```

Comment:
NB: Numbers from 1 and the positive difference between 7&123 (ie find the positive difference and print the numbers between 1 and that number while fulfilling other criteria)